src/crowdsale/CrowdSale.sol

```diff
@@ -74,18 +74,16 @@ contract CrowdSale is ReentrancyGuard {
      * @return saleId
      */
     function startSale(Sale calldata sale) public returns (uint256 saleId) {
-        //todo: use 15 minutes
-        if (sale.closingTime < block.timestamp + 2 hours) {
+        if (sale.closingTime < block.timestamp) {
```

Comment:
You can have an instantly closing sale now.
The `+ 2 hours` was a not bad solution

Reply:
yeah, we talked about that quite some time. Imo the team "wants" it, even though its pointless (it makes sales simpler to test in test & stage envs but imo we should rather preprocess / code configure this restricion). But what actually matters most here: I don't think that restricting the closing time breaks anything but user experience.